AdyenCard/Validators/CardAddressValidator.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import Foundation
+
+/// Validates a card's address field.
+public final class CardAddressValidator {
```

Comment:
Card component is not the only place having addresses so let's rename it into AddressValidator.
I'm assuming we will need this for other addresses we have.

Reply:
Good point. I will change this.